### PR TITLE
Move balances to async frame on event index

### PR DESCRIPTION
**Pull request by maxwofford**
This balance just shows up for users of beta "detailed_events_list_2022_10_04", but can be pretty heavy

### CODE REVIEW

File: app/views/events/balance.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<span class="no-wrap <%= 'muted' if @balance.zero? %>">
+    <%= inline_icon 'payment', size: 16, class: 'muted align-middle', 'aria-label': 'Dollar sign', style: 'transform: scale(2)' %>
```
Comment (cjdenio) [inline app/views/events/balance.html.erb]:
```suggestion
  <%= inline_icon 'payment', size: 16, class: 'muted align-middle', 'aria-label': 'Dollar sign', style: 'transform: scale(2)' %>
```

File: app/views/static_pages/index.html.erb [hunk 2]
```diff
@@ -41,11 +41,13 @@
               <strong class="brand-sans h3 line-height-2 mb1"><%= event.name %></strong>
               <% if Flipper.enabled?(:detailed_events_list_2022_10_04, current_user) %>
                 <div class="brand-sans h4 line-height-2 mb2 flex justify-between"> 
+                  <%= async_frame_to event_balance_path(event_id: event.id), as: :span do %>
```
Comment (cjdenio) [inline app/views/static_pages/index.html.erb]: You might consider using a [Turbo Frame](https://turbo.hotwired.dev/handbook/frames#eager-loading-frames) cuz they're cool (and I believe Rails automatically strips the layout from Turbo Frame requests)

**Review by cjdenio — APPROVED** [review]

**Review by YodaLightsabr — APPROVED** [review]
Looks good to me! #3096 seems to also do this with `async_balance` instead; maybe we could copy over the logo code and close that PR?

**Comment by maxwofford** [comment]
@YodaLightsabr oh heck yes! I'll close this in favor of that.

**Comment by maxwofford** [comment]
@YodaLightsabr can we merge #3096 so people can test it out? cc @cjdenio

**Pull request closed without merging**